### PR TITLE
Updated structure for CD docs

### PR DESCRIPTION
**Pull request by swoodward90**
## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: pages/deployments/browser-quickstart.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+{% embed url="https://www.youtube.com/watch?v=jBlq45ntbBw" aspectRatio="16 / 9" /%}
+
+## Onboard to Plural and install the Plural Console
+
+If you haven't already, you'll need to follow the Plural guide to install Console. You can use the guide for the [in-browser Cloud Shell](/getting-started/cloud-shell-quickstart) or the [CLI](/getting-started/quickstart) to get started.
+
+## Log in to Console and Add Clusters
+
+Once you've logged in to Plural Console, you can click the "Continuous deployment" tab to set up your clusters. By default, you should see the cluster that your Console is deployed on.
```
Comment (swoodward90) [inline pages/deployments/browser-quickstart.md:25]: @michaeljguarino just checking that this is true, they'll always at least have their current console cluster visible here

File: pages/deployments/browser-quickstart.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+To deploy additional clusters, click the "Create cluster" button in the top right. To spin up a new cluster, fill in the required fields:
+
+![](/assets/deployments/create-cluster.png)
+
+To import an existing cluster, select the "Import existing" toggle and add a name, handle, and any tags. You then need to run the following command on the cluster itself to install the Plural Deploy Operator:
```
Comment (swoodward90) [inline pages/deployments/browser-quickstart.md:31]: @michaeljguarino any additional detail here on where to run this command? I can add this to the BYOK section too if helpful

File: pages/deployments/browser-quickstart.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+```
+plural cd install --url https://<CONSOLE_URL>/ext/gql -- token <TOKEN>
```
Comment (swoodward90) [inline pages/deployments/browser-quickstart.md:34]: @michaeljguarino this is just the token from Console right? I can add instructions on how to get it or link to that helper page if so

Comment (swoodward90) [reply pages/deployments/browser-quickstart.md:34]: will also remove space in front of token

File: pages/deployments/global-services.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
+title: Configure Global Services
+description: Deploy services by default on specified clusters.
+---
+
+Global services are automatically replicated across existing and new clusters matching the set of defined specifications.
+
+Once a service is deployed onto any cluster, it can be marked global by clicking the three dots in the deployments table and selecting "Make global".
```
Comment (swoodward90) [inline pages/deployments/global-services.md:8]: @michaeljguarino what's the CLI command for this?

File: pages/deployments/upgrades.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+---
+title: Upgrades and Rollbacks
+description: Change versions of deployed services
+---
+
+## Upgrades
+
+Plural will track new revisions for a service when the underlying Git repo or configured secrets are changed. When PRs are merged to the tracked branch in Git, the service will be updated after the next pull. Modifying secrets will also trigger an update.
```
Comment (swoodward90) [inline pages/deployments/upgrades.md:8]: @michaeljguarino confirm correctness

File: pages/deployments/browser-quickstart.md (hunk 1 above)
Comment (michaeljguarino) [reply pages/deployments/browser-quickstart.md:25]: yep

File: pages/adding-new-application/plural-artifact-structure.md [hunk 6]
```diff
@@ -85,7 +86,6 @@ $ tree .
 
 Let's disect this artifact's structure.
```
Comment (dogmar) [inline pages/adding-new-application/plural-artifact-structure.md:87]:
```suggestion
Let's dissect this artifact's structure.
```

File: pages/deployments/browser-quickstart.md (hunk 3 above)
Comment (michaeljguarino) [reply pages/deployments/browser-quickstart.md]: I think the ui should tell them exactly what to run, yea

File: pages/deployments/global-services.md (hunk 4 above)
Comment (michaeljguarino) [reply pages/deployments/global-services.md:8]: not sure we have one yet

**Review by michaeljguarino — APPROVED** [review]

**Pull request merged**